docs/usage/self-hosted-configuration.md

```diff
@@ -1140,6 +1150,18 @@ The combinations of `requireConfig` and `onboarding` are:
 | `requireConfig=optional` | An onboarding PR will be created if no config file exists. If the onboarding PR is closed and there's no config file, the repository will be processed. | Repository is processed regardless of config file presence.   |
 | `requireConfig=ignored`  | No onboarding PR will be created and repo will be processed while ignoring any config file present.                                                     | Repository is processed, any config file is ignored.          |
 
+## s3Endpoint
+
+If set, Renovate will use this string as the `endpoint` when creating the AWS S3 client instance.
+
+## s3PathStyle
+
+If set, Renovate will enable `forcePathStyle` when creating the AWS S3 client instance.
+
+Quote from [AWS S3 documentation - Interface BucketEndpointInputConfig](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/bucketendpointinputconfig.html):
+
+> Whether to force path-style URLs for S3 objects (e.g., `https://s3.amazonaws.com//` instead of `https://.s3.amazonaws.com/`)
```

Comment:
```suggestion
For example:

| `s3PathStyle` | Path                               |
| ------------- | ---------------------------------- |
| `off`         | `https://bucket.s3.amazonaws.com/` |
| `on`          | `https://s3.amazonaws.com/bucket/` |

Read the [AWS S3 docs, Interface BucketEndpointInputConfig](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/interfaces/bucketendpointinputconfig.html) to learn more about path-style URLs.
```

I like @viceice's suggestion above. But quotes must match the actual quote.

How about this instead?

- Use a table to show the difference between config option on/off
- Drop the AWS docs quote
- Rewrite sentence to mention the AWS docs

My suggestion only gives you the _layout_. Please update the table so it shows the _correct_ information!